hyperon_das/api.py

```diff
@@ -26,44 +31,78 @@
     QueryAnswer,
     QueryOutputFormat,
     QueryParameters,
+    config,
 )
 
 
 class DistributedAtomSpace:
     def __init__(
-        self,
-        database: DatabaseType,
-        host: Optional[str] = None,
-        port: Optional[str] = None,
+        self, das_type: DasType = DasType.CLIENT.value, **kwargs
     ) -> None:
-        self._db_type = database
+        self._type = das_type
+        self.__remote_das = []
 
         try:
-            DatabaseType(database)
+            DasType(self._type)
         except ValueError as e:
             self._error(
-                DatabaseTypeException(
+                DasTypeException(
                     message=str(e),
-                    details=f'possible values {DatabaseType.values()}',
+                    details=f'possible values {DasType.types()}',
                 )
             )
 
         logger().debug(
             {
                 'message': '[DistributedAtomSpace][__init__]',
-                'data': {'database_type': database},
+                'data': {'das_type': das_type},
             }
         )
 
-        if database == DatabaseType.SERVER.value and not host:
-            raise InitializeServerException(
-                message='You must send the host parameter',
-                details=f'To use server type Das you must send at least the host parameter',
-            )
-
-        self.db = database_factory(DatabaseFactory(self._db_type), host, port)
+        if self._type == DasType.CLIENT.value:
+            self.db = InMemoryDB()
+        elif self._type == DasType.SERVER.value:
+            # TODO: Implement some like that in the future
```

Comment:
Please remove this comment and add a card in the board instead. This should be the standard procedure for TODOs. In this project.

Reply:
Ok